server/routes/trivy.ts

```diff
@@ -0,0 +1,130 @@
+import { type RequestHandler, Router } from 'express'
+import asyncMiddleware from '../middleware/asyncMiddleware'
+import type { Services } from '../services'
+import { isValidDropDown } from '../utils/utils'
+
+export default function routes({ componentNameService, dataFilterService }: Services): Router {
+  const router = Router()
+
+  const get = (path: string, handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
+  // const post = (path: string, handler: RequestHandler) => router.post(path, asyncMiddleware(handler))
+
+  get('/', async (req, res) => {
+    if (req.query.updateServiceArea === '' && isValidDropDown(req, 'serviceArea')) {
```

Comment:
do we actually need this query parsing handling? 

I think e.preventDefault would mean these are never sent

Reply:
preventDefault just stops the default action firing I believe https://api.jquery.com/event.preventDefault/

Reply:
Its sorted by https://github.com/ministryofjustice/hmpps-developer-portal/pull/227 which is merged in this PR .